src/filters/reframer.c

```diff
@@ -2774,6 +2787,7 @@ static const GF_FilterArgs ReframerArgs[] =
 	"- no: do no filter frames based on cue info\n"
 	"- segs: only forward frames marked as segment start\n"
 	"- frags: only forward frames marked as fragment start", GF_PROP_UINT, "no", "no|segs|frags", GF_FS_ARG_HINT_EXPERT|GF_FS_ARG_UPDATE},
+	{ OFFS(sapcue), "treat SAPs as cue points", GF_PROP_BOOL, "false", NULL, GF_FS_ARG_HINT_EXPERT },
```

Comment:
this should not be a boolean but rather a SAP value: 0 (default) disables, otherwise only set cue if sap>0 and sap<=sapcue